package-lock.json

```diff
@@ -7442,25 +7452,25 @@
       "dependencies": {
         "abbrev": {
           "version": "1.1.1",
-          "resolved": "https://registry.npmjs.org/abbrev/-/abbrev-1.1.1.tgz",
+          "resolved": "",
```

Comment:
:-1:

Reply:
Do not install dependencies without having done a `npm ci` just before.
You can try `npx fix-lock package-lock.json` to avoid having to restart installation from beginning.